### DISCUSSION TITLE
Script delay

### DISCUSSION THREAD

**Discussion by boromyr** [AutoDarkMode/Windows-Auto-Night-Mode#810] [Q&A]
Is it possible to add a delay for the execution of the scripts? For example, run script1, Sleep 1000, run script2?

**Comment by envico801** [comment]
Hi @boromyr, I don't know if it's possible, but it occurs to me that maybe integrating a delay from the script itself could work.

A possible solution could be to create a .bat file that runs your script with a delay.

`test.bat`

```batch
timeout /t 5 && your_script_goes_here
```

e.g.

```batch
timeout /t 5 && node path/to/script.js
```

---

Or maybe using a library from your script itself.

`myScript.js`

e.g.

```javascript
setTimeout(run_code(), 5000);
```
